app/base.scss

```diff
@@ -0,0 +1,61 @@
+*{
```

Comment:
**TODO FOR LOST POINTS**

Your ``*`` selector needs to include the following attributes per lab instructions:

- clear padding and margins
- set everything to be width 100%
- set font size, weight
- clear form focus
- clear list item style
- clear text-decoration
- set font color